.github/workflows/int_test.yml

```diff
@@ -144,15 +223,16 @@ jobs:
           az iot ops mq stats
           az iot ops mq stats --raw
           az iot ops mq get-password-hash -p test
-          az iot ops asset query -g ${{ env.CLUSTER_RG }}
+          az iot ops asset query -g ${{ env.RESOURCE_GROUP }}
           az iot ops verify-host
+
   # Optional cleanup job
   cleanup:
-    needs: [integration_test]
+    needs: [test]
     if: ${{ github.event.inputs.cleanup == 'true' }}
     uses: './.github/workflows/cluster_cleanup.yml'
     with:
-      cluster_prefix: ${{ needs.integration_test.outputs.cluster_name }}
-      resource_group: ${{ needs.integration_test.outputs.resource_group }}
-      keyvault_prefix: ${{ needs.integration_test.outputs.keyvault_name }}
+      cluster_prefix: "az-iot-ops-test-cluster${{ github.run_number }}"
+      resource_group: "${{ inputs.resource_group }}"
+      keyvault_prefix: "opskv${{ github.run_number }}"
```

Comment:
we have `env:` defined above with the same values for RG and keyvault prefix right?
![image](https://github.com/Azure/azure-iot-ops-cli-extension/assets/28658112/c641dfac-2964-4862-8d16-b7a7ac02b0ad)

Is there value in referencing the environment variables here for consistency ?

Reply:
There would be, but github does not allow referencing `env` in `env` blocks.

There's a solution for using job outputs to pass-through environment variables, but with matrix jobs it's more convoluted to manage multiple outputs